apps/backend-responses/AddResponse/index.ts

```diff
@@ -68,21 +74,10 @@ const httpTrigger: AzureFunction = async function (
 
   const redisMulti = redisExec.pipeline()
 
-  // if we are processing a first response, set the timestamp on the instance
-  // this will allow us to award points for response timing
-  // TODO: do this on the first correct response only
-  if (!firstResponseReceivedAt) {
```

Comment:
moved this so that it is set on the first correct response, not the first ever. the logic below is not DRY yet and can be cleaned up a bit